src/specifications/profile-resolution/profile-resolution-specml.xml

```diff
@@ -3,1772 +3,1370 @@
+          impact baselines HIGH, MODERATE and LOW: each of these calls on a catalog representing the
+          families, controls and control enhancements described in SP 800-53/53A. In a data
+          processing system, this dependency can be modeled in the form of separate documents or
+          data instances - the (nominal) <q>catalog</q> and its <q>profile</q>, the latter being a
+          selection, adaptation and organization of controls, as distinct from the catalog from
+          which those controls are derived.</p>
+        <p>However, profiles may also select controls from profiles, thus deriving them from
+          underlying catalogs as modified by those profiles. This feature enables an organization
+          that wishes to promulgate guidelines or requirements to do so by publishing a set of
+          customizations to an existing catalog, rather than create a catalog from scratch. Profiles
+          can also arrange combinations of controls from more than one catalog or profile origin,
+          making it possible to use them to support the creation of hybrid catalogs. </p>
+        <p>In the general case OSCAL must support sourcing more than one catalog at once, in order
+          to aggregate and organize controls from disparate sources. For example, it should be
+          routine for a profile to acquire its controls from an authoritative standard catalog, plus
+          a local supplement. But multiple catalog inputs, especially when they are produced and
+          sourced independently of one another, may not combine without issues. In order to provide
+          sensible and transparent results even for incoherent inputs, this specification defines
+          these processes such that legible and testable results are produced even when they are
+          formally invalid to constraints governing the outputs. Such downstream processing errors
+          including validation errors are expected to be detectable and remediable. This especially
+          applies to situations where multiple catalogs are sourced, or where the same catalog is
+          sourced more than once, directly or indirectly.</p>
+        <p>In order to support these semantics – because whatever data set is sourced by a profile,
+          it must <q>look like a catalog</q> – the results of resolving a single profile must be in
+          the form of a single catalog. Two different OSCAL processors that promise profile
+          resolution, must deliver, for the same input, the same catalog with regard to its
+          contents: which controls are included; their ordering (insofar as the target format
+          represents ordering); and the structure (groups) in which they are embedded. This
+          specification is designed to make this possible, by defining outputs with sufficient rigor
+          that <q>the same</q> is meaningful and testable in this context.</p>
```

Comment:
As I understand @stephenbanghart hasn't done much with this section yet? Agree it could be flattened and cut. One or two bits (such as how outputs may not be formally valid to OSCAL for bad inputs) could be pulled to elsewhere? Other bits might already be stated elsewhere, or not worth stating.

Also with DW wonder if the header is right for this section at least as it stands. Basically I think it says "btw because of the way catalogs work in the wild, profile resolution cannot guarantee OSCAL valid outputs for any arbitrary combination of valid OSCAL inputs (catalogs and/or upstream profiles). I think that is correct but not sure where/how it should be stated.

Reply:
@wendellpiez Yes. I believe what you are stating above is correct.